### PR TITLE
[meta-aos-vm] Don't update time during node encryption

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: meta-aos-vm-main/recipes-aos/aos-iamanager/files/aos_iamanager.cfg [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "DiskEncryptionCmdArgs": [
         "/bin/sh",
         "-c",
-        "/opt/aos/setupdisk.sh create $(cat /proc/cmdline | sed -e 's/^.*opendisk.target=//' -e 's/ .*$//') -m /usr/lib/softhsm/libsofthsm2.so -p $(cat /var/aos/iam/.usrpin) 2>&1 | systemd-cat; systemctl restart nfs-server.service || true"
+        "systemctl stop systemd-timesyncd ;/opt/aos/setupdisk.sh create $(cat /proc/cmdline | sed -e 's/^.*opendisk.target=//' -e 's/ .*$//') -m /usr/lib/softhsm/libsofthsm2.so -p $(cat /var/aos/iam/.usrpin) 2>&1 | systemd-cat;systemctl start systemd-timesyncd ;systemctl restart nfs-server.service || true"
```
Comment (al1img) [inline meta-aos-vm-main/recipes-aos/aos-iamanager/files/aos_iamanager.cfg:51]: `; `

File: meta-aos-vm-node/recipes-aos/aos-iamanager/files/aos_iamanager.cfg [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "DiskEncryptionCmdArgs": [
         "/bin/sh",
         "-c",
-        "/opt/aos/setupdisk.sh create $(cat /proc/cmdline | sed -e 's/^.*opendisk.target=//' -e 's/ .*$//') -m /usr/lib/softhsm/libsofthsm2.so -p $(cat /var/aos/iam/.usrpin) 2>&1 | systemd-cat"
+        "systemctl stop systemd-timesyncd ;/opt/aos/setupdisk.sh create $(cat /proc/cmdline | sed -e 's/^.*opendisk.target=//' -e 's/ .*$//') -m /usr/lib/softhsm/libsofthsm2.so -p $(cat /var/aos/iam/.usrpin) 2>&1 | systemd-cat ;systemctl start systemd-timesyncd"
```
Comment (al1img) [inline meta-aos-vm-node/recipes-aos/aos-iamanager/files/aos_iamanager.cfg:36]: ditto

File: meta-aos-vm-main/recipes-aos/aos-iamanager/files/aos_iamanager.cfg (hunk 1 above)
Comment (mykola-kobets-epam) [reply meta-aos-vm-main/recipes-aos/aos-iamanager/files/aos_iamanager.cfg]: done

File: meta-aos-vm-node/recipes-aos/aos-iamanager/files/aos_iamanager.cfg (hunk 2 above)
Comment (mykola-kobets-epam) [reply meta-aos-vm-node/recipes-aos/aos-iamanager/files/aos_iamanager.cfg]: done

**Review by mlohvynenko — APPROVED** [review]

**Comment by mykola-kobets-epam** [comment]
not required

**Pull request closed without merging**